### DISCUSSION TITLE
2023년 학습로그 연간 계획

### DISCUSSION THREAD

**Discussion by pocojang** [woowacourse/prolog#1092] [Ideas]
### Prolog 2.0?
>스쿼드로 나눠서 진행

#### 1. 로드맵

- 목표
  - 자기주도 학습 방법을 만든다 (시간, 공간 제약 없이)
  - 데이터 기반으로 학습이 필요한 부분을 파악할 수 있도록 한다.
- 핵심 지표
  - 로드맵 달성률? (강사 입장, 크루 입장)
    - 역량 체크, 학습 수행도
      - 예) 강사가 제시하는 달성률을 만족하는 크루들을 볼 수 있다.
      - 예) 필수 로드맵을 제시하고 잘 학습했는지 달성률로 검증할 수 있다.
  - 레벨 인터뷰와 연계할 수 있는 방법 모색

#### 2. 외부 노출 컨텐츠 (네이밍)

- 예시
  - [서핏](https://www.surfit.io)
  - [Disquiet](https://disquiet.io)
  - [Velog](https://velog.io)
- 목표
  - 일반(외부) 사용자들과 함께 할 수 있는 서비스 / 학습 컨텐츠 커뮤니티
    - 외부 노출(UI 개선)과 컨텐츠 추상화를 나눌 수 있을까?
    - 언젠가는 외부인이 컨텐츠를 생산한다?
  - 우테코에서 생산된 다양한 컨텐츠를 구독하고 관리할 수 있다.
    - 컨텐츠를 관리하는 부분은 로드맵 스쿼드와 연계할 수 있을까?
- 핵심 지표
  - 외부 가입자 비율을 늘린다.
  - 학습 로그가 아닌 다른 컨텐츠 비율도 N% 달성한다.
  - GA 등의 도구를 활용하여 MAU, DAU 등의 지표를 활성화한다.
  - A/B 테스트를 통해 지표를 분석한다.

---

## 다음 논의

- [ ] 컨텐츠 추상화에 대한 의존성과 방향 추가 논의 해보기
- [ ] 리소스 분배 방법에 대한 이야기
  - 우테코가 종료되어도 할 수 있는 크루들에 대한 취합과 방법
- [ ] 로드맵 달성률 체크 방법 / 다음 달에 커리큘럼 담당 코치들 정해질 경우 추가 논의 해보기
- [ ] OKR
- [ ] 스쿼드별 이상적인 근로 인원 수

## Backlog

- [ ] 로드맵 달성률 체크 방법 (2023년 중반?)

**Comment by gracefulBrown** [comment]
# 논의 할 내용
- [x] 내년도 상세 계획 추가 논의
  - [x] 로드맵과 아티클(컨텐츠 추상화)
  - [x] 사용자 확보 및 외부 오픈
- [x] 내년 근로 시작 전 해야 할 일 논의
  - 서비스 기획 정리
  - 백엔드 인프라 정리
  - 코드 정리 (백엔드)
  - 문서 정리 (프론트엔드)
  - 개발 프로세스 정리
    - OKR?
    - 스쿼드 인원
    - 달성률 체크 방법

# 2022년 일정
- [ ] 서비스 기획 정리
  - [x] [기존 기능 논의](https://github.com/woowacourse/prolog/discussions/1105) - 토미
  - [ ] 아티클 관련 기획 논의 - 브라운
- [ ] [학습로그, 레벨로그, 레벨 인터뷰 제도 검토 및 개선 논의](https://github.com/woowacourse-projects/techcourse/issues/1500)
- [ ] 문서와 피그마 정리 - 포코
- [ ] 수료 후 크루들과의 협업 일정 논의 - 마지막 주

# 기타 일정
- [ ] 12월 12일 로드맵 기능 근로 내 공유 및 실 공유 일정 계획 논의

**Reply by gracefulBrown** [reply]
## 아티클
- https://github.com/woowacourse/prolog/discussions/1121

## 로드맵
- 어드민 동작해야 함 - @gracefulBrown 
- 현재 작업 진행중인 것이 마무리되어야 함 - @seokhongkim 
- 질답 형식으로 글 남길 수 있게 하기 - @seokhongkim 

## 기타
- 칸반 보드 구성 - @gracefulBrown

**Comment by gracefulBrown** [comment]
## 1월중에 할(해야하는) 일
- 학습로그 프로세스 및 제도 개선 - 12월 20일
- 문서화 정리 
  - 백엔드 문서와 프론트 문서 분리된 상태
  - 각 문서들을 보기 좋게 모아놓기
- 로드맵 기능 마무리
- 내년 레벨1 전까지 질답형식 글 작성

## 홀딩
- 아티클 기능 프로토타입 화면 
- 코드, 서비스 정리 
- 인프라 정리
- (어드민)리액트 어드민 제거 검토
